### PR TITLE
Enrich notifications

### PR DESCRIPTION
**Pull request by upacyxou**
- enrich notifications

### CODE REVIEW

File: src/helpers/env.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default cleanEnv(process.env, {
   BUNDLE_ID: str(),
   ETH_NETWORK: str({ default: ETH_MUMBAI_NETWORK }),
   ETH_RPC: str({ default: BWL_ETH_RPC }),
+  IPFS_GATEWAY: str({ default: 'https://ipfs.sealcred.xyz/ipfs' }),
```
Comment (T-Damer) [inline src/helpers/env.ts:20]: Hm, didn't we have this one in `bwl/env`?

Comment (T-Damer) [reply src/helpers/env.ts:20]: If we don't, we should add, I think

**Review by smyzin — APPROVED** [review]

File: src/helpers/getIPFSContent.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+import env from '@/helpers/env'
+
+export default async function (cid?: string) {
+  if (!cid) return
+  const ipfsBaseUrl = env.IPFS_GATEWAY
+  const ipfsUrlString = `${ipfsBaseUrl}/${cid}`
+  const ipfsContent = (await axios.get<{ text?: string }>(ipfsUrlString)).data
```
Comment (T-Damer) [inline src/helpers/getIPFSContent.ts:8]: How long will it wait before canceling the request? I remember `fetch` waited for too long, we need something like 10-15s

File: src/helpers/sendFirebaseNotification.ts [hunk 3]
```diff
@@ -3,7 +3,12 @@ import firebase from '@/helpers/firebase'
 
 const messaging = getMessaging(firebase)
 
-export default function (token: string, title?: string, postId?: number) {
+export default function (
```
Comment (T-Damer) [inline src/helpers/sendFirebaseNotification.ts:6]: I think we should do an object for props there

File: src/helpers/getIPFSContent.ts (hunk 2 above)
Comment (smyzin) [reply src/helpers/getIPFSContent.ts:8]: Do you think it can take more than 1-2 seconds?

File: src/helpers/env.ts (hunk 1 above)
Comment (upacyxou) [reply src/helpers/env.ts:20]: no we don't have it

File: src/helpers/getIPFSContent.ts (hunk 2 above)
Comment (upacyxou) [reply src/helpers/getIPFSContent.ts:8]: 5 seconds

**Pull request merged**